code/espurna/libs/HeapStats.h

```diff
@@ -6,51 +6,36 @@ Show extended heap stats when EspClass::getHeapStats() is available
 
 #pragma once
 
-#include <type_traits>
+#include "TypeChecks.h"
```

Comment:
And general question - TypeTraits.h?

Reply:
As long as it's only an ``is_detected`` implementation, I guess it's mostly type checks/type detections.